docs/metrics.md

```diff
@@ -1,17 +1,31 @@
-# SSP Operator metrics
-This document aims to help users that are not familiar with metrics exposed by the SSP Operator.
-All metrics documented here are auto-generated by the utility tool `tools/metricsdocs` and reflects exactly what is being exposed.
+# SSP Operator Metrics
+
+This document helps users unfamiliar with SSP Operator metrics.
+
+**Note:** All listed metrics are auto-generated by `tools/metricsdocs`, reflecting the exposed metrics accurately.
+
+## Metrics
 
-## SSP Operator Metrics List
```

Comment:
I know it is not the best of approaches but we have some automation (https://github.com/kubevirt/monitoring/blob/666a752cedc63ee746060fa805c406d5a93abf83/tools/metricsdocs/metricsdocs.go#L185) that is tracking this title

If you want to keep this change you proposed, I would suggest adding a comment to identify the beginning and the end of the metrics list here and submit a patch to https://github.com/kubevirt/monitoring to support both approaches

Reply:
Okay, thanks. I'll not change the `metrics.md` in this PR, and revert my changes.